File: Mobile/objects/requests/Orders/CreateShippingOrderRequestModel.ts
import { AddressRequestModel } from "../Address/AddressRequestModel";
import { CreateOrderDetailsRequestModel } from "../OrderDetails/CreateOrderDetailsRequestModel";

export interface CreateShippingOrderRequestModel {
    campaignId?: number;
    addressRequest: AddressRequestModel;
    freight?: number;
    payment: number;
    orderDetails: CreateOrderDetailsRequestModel[];
}

File: BE/Boek.Infrastructure/Requests/Orders/CreateShippingOrderRequestModel.cs
using System.Text.Json.Serialization;
using Boek.Infrastructure.Requests.Addresses;
using Boek.Infrastructure.Requests.OrderDetails;

namespace Boek.Infrastructure.Requests.Orders
{
    public class CreateShippingOrderRequestModel
    {
        public int? CampaignId { get; set; }
        public AddressRequestModel AddressRequest { get; set; }
        [JsonIgnore]
        public string Address { get; set; }
        public decimal? Freight { get; set; }
        public byte Payment { get; set; }

        public List<CreateOrderDetailsRequestModel> OrderDetails { get; set; }
    }
}